### PR TITLE
feat: add ApiClient#getEvaluations

### PR DESCRIPTION
**Pull request by yshrsmz**

### CODE REVIEW

File: src/internal/model/SourceID.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const SourceID = {
+  UNKNOWN: 0,
+  ANDROID: 1,
+  IOS: 2,
+  WEB: 3,
+  GOAL_BATCH: 4,
+  GO_SERVER: 5,
+  NODE_SERVER: 6,
```
Comment (yshrsmz) [inline src/internal/model/SourceID.ts:8]: @cre8ivejp Is it ok to assume that SourceID is still a number type?

Comment (cre8ivejp) [reply src/internal/model/SourceID.ts:8]: Yes.
https://github.com/bucketeer-io/bucketeer/blob/main/proto/event/client/event.proto#L61-L69

File: src/internal/remote/ApiClient.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  async getEvaluations(
+    user: User,
+    userEvaluationsId: string,
+    timeoutMillis: number = this.defaultRequestTimeoutMillis
+  ): Promise<GetEvaluationsResult> {
+    const body: GetEvaluationsRequest = {
+      tag: this.featureTag,
+      user,
+      user_evaluations_id: userEvaluationsId,
+      source_id: SourceID.ANDROID
```
Comment (cre8ivejp) [inline src/internal/remote/ApiClient.ts:37]: Please change it to `SourceID.WEB`

File: src/internal/remote/toBKTException.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { ApiServerException, BadRequestException, BKTException, FeatureNotFoundException, InvalidHttpMethodException, UnauthorizedException, UnknownException } from '../../BKTExceptions'
+import { ErrorResponse } from '../model/response/ErrorResponse'
+import { FetchResponseLike } from './fetch'
+
+export const toBKTException = async (response: FetchResponseLike): Promise<BKTException> => {
+  const errorBody: ErrorResponse | null = await response.json()
+  const error = errorBody?.error
+
+  switch (response.status) {
```
Comment (cre8ivejp) [inline src/internal/remote/toBKTException.ts:9]: The server errors have changed to the following.
https://github.com/bucketeer-io/android-client-sdk/issues/56

Let me know if you prefer to do this in another PR.

File: src/internal/remote/ApiClient.ts (hunk 2 above)
Comment (yshrsmz) [reply src/internal/remote/ApiClient.ts]: [`7502934` (#13)](https://github.com/bucketeer-io/javascript-client-sdk/pull/13/commits/7502934112f931f98b9a23538c91490e954b1f1e)

File: src/internal/remote/toBKTException.ts (hunk 3 above)
Comment (yshrsmz) [reply src/internal/remote/toBKTException.ts:9]: got it. updated in [`801b304` (#13)](https://github.com/bucketeer-io/javascript-client-sdk/pull/13/commits/801b304e9fd9f8649b874a473149be55e78f1b39)

**Review by cre8ivejp — APPROVED** [review]
Thank you!!
